src/styles/pages/plp.scss

```diff
@@ -0,0 +1,30 @@
+// Product
```

Comment:
On the first load of [the PLP](https://preview-480--base.preview.vtex.app/office) we're missing the shelf styles:
<img src="https://user-images.githubusercontent.com/381395/163295792-63338b02-e720-486d-a081-8037043f8142.png" width="50%">

Reply:
Tested now and it's been resolved!